File: core/src/backendServer/models/Bazooka.js
const constants = require('../settings/constants');
const Weapon = require('./Weapon');
const ProjectileType = require('./ProjectileType');

class Bazooka extends Weapon {
    constructor() {
        super(constants.BAZOOKA_RELOAD_TIME, constants.BAZOOKA_BULLETS_IN_CHAMBER,
            constants.BAZOOKA_MAGAZINE_RELOAD_TIME, ProjectileType.BAZOOKA)
    }
}

module.exports = Bazooka;

File: core/src/com/mygdx/game/model/weapon/Bazooka.kt
package com.mygdx.game.model.weapon

import com.mygdx.game.model.projectile.ProjectileType
import com.mygdx.game.settings.BAZOOKA_BULLETS_IN_CHAMBER
import com.mygdx.game.settings.BAZOOKA_RELOAD_TIME

class Bazooka : Weapon(BAZOOKA_RELOAD_TIME, BAZOOKA_BULLETS_IN_CHAMBER, ProjectileType.BAZOOKA)